Sources/BoogieWrapper/source/Wrapper.cs

```diff
@@ -32,7 +35,7 @@ static int Main(string[] args)
             string funcName = args[1];
 
             //TODO: Make it aware of the other Boogie options
-            var boogieOptions = " -doModSetAnalysis -printInstrumented -z3multipleErrors -typeEncoding:m -timeLimit:" + Options.Timeout + " -removeEmptyBlocks:0 -printModel:1 -printModelToFile:model.dmp " + Options.BoogieUserOpts;
+            var boogieOptions = " -doModSetAnalysis -printInstrumented -monomorphize -timeLimit:" + Options.Timeout + " -removeEmptyBlocks:0 -printModel:1 -printModelToFile:model.dmp " + Options.BoogieUserOpts;
```

Comment:
It might be worth mentioning the reasoning for these changes in the PR description.

`-typeEncoding:m` seems to have disappeared from CLI options (but remains available to set programmatically) sometime before Boogie v2.9.1, before resurfacing again after Boogie v3. `-monomorphize` attempts monomorphization, and throws an error if that fails, which should achieve the same.

`-z3multipleErrors` was completely removed in [Boogie #197](https://github.com/boogie-org/boogie/pull/197), and `MULTI_TRACES` was removed in [Boogie #201](https://github.com/boogie-org/boogie/pull/201).

Reply:
As a result of losing `-z3multipleErrors`, SymDiff option `useMultipleGoodRuns` was removed. Maybe it could be added back if there is a workaround for getting multiple errors again, but I could not find one on a quick search.

Reply:
Thanks for describing these differences! I'll use your text to update the description. :)